src/test/resources/application-test.yml

```diff
@@ -19,36 +19,36 @@ hmpps.sqs:
   provider: localstack
   queues:
     visit:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
+      queueName: "visit-${random.uuid}"
+      dlqName: "visit-dlq-${random.uuid}"
       subscribeTopicId: domainevents
       subscribeFilter: "{\"eventType\":[\"prison-visit.booked\", \"prison-visit.changed\", \"prison-visit.cancelled\"]}"
       dlqMaxReceiveCount: 1
       visibilityTimeout: 1
     incentive:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
+      queueName: "incentive-${random.uuid}"
+      dlqName: "incentive-dlq-${random.uuid}"
       subscribeTopicId: domainevents
       subscribeFilter: "{\"eventType\":[\"incentives.iep-review.inserted\", \"incentives.level.changed\", \"incentives.levels.reordered\", \"incentives.prison-level.changed\"]}"
       dlqMaxReceiveCount: 2
       visibilityTimeout: 1
     activity:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
+      queueName: "activity-${random.uuid}"
+      dlqName: "activity-dlq-${random.uuid}"
       subscribeTopicId: domainevents
       subscribeFilter: "{\"eventType\":[\"activities.activity-schedule.created\", \"activities.activity-schedule.amended\", \"activities.scheduled-instances.amended\", \"activities.scheduled-instance.amended\", \"activities.prisoner.allocated\", \"activities.prisoner.deallocated\", \"activities.prisoner.allocation-amended\", \"activities.prisoner.attendance-created\", \"activities.prisoner.attendance-amended\"]}"
       dlqMaxReceiveCount: 1
-      visibilityTimeout: 1
+      visibilityTimeout: 120
```

Comment:
This caused me so much pain! A visibility timeout of 1 means that if the tests run slowly for some reason Localstack puts messages that are actually in progress back on the queue - hence the messages are processed twice and tests fail in seemingly random ways.

Reply:
:-)